TCAT/Utilities/Styles.swift

```diff
@@ -46,10 +74,35 @@ extension UIFont {
     
     /// Generate fonts for app usage
     static func style(_ name: String, size: CGFloat) -> UIFont {
```

Comment:
can we call this method `font` or `getFont` instead of `style` since function returns a `UIFont`

Reply:
also think it reads better if have "name:" argument show up as a parameter label when use function (i.e. `name:` instead of `_ name:`)

Reply:
The function is technically UIFont.style I used `_ name:` because I figured it would be obvious what the parameter means, and would save space. Do you feel strongly about this? I don't.

Reply:
I changed the name, kept the first param